erpnext/regional/address_template/templates/bosnia_herzegovina.html

```
@@ -0,0 +1,4 @@
+{{ address_line1 }}<br>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)